platform/qt/include/qmapboxgl.hpp

```diff
@@ -232,15 +225,29 @@ public slots:
 
 signals:
     void needsRendering();
-    void mapChanged(QMapboxGL::MapChange);
     void copyrightsChanged(const QString &copyrightsHtml);
 
+signals:
+    void cameraWillChange(QMapboxGL::CameraChangeMode);
```

Comment:
I'm not sure if I like this. For the high level API we don't need these signals anyway...

Reply:
@tmpsantos based on the comments above, can we revisit this? I'm also :+1: in finding other ways for passing the arguments for the signals mentioned.

Reply:
Alright, but we need to add documentation for all these signals :smile:

Reply:
I think this PR was before we add docs for everything...